Adjust for changed UI search and generation paths in CMake 3.9.

--- CMakeLists.txt.orig	2017-05-09 18:52:59 UTC
+++ CMakeLists.txt
@@ -24,6 +24,10 @@ CONFIGURE_FILE(Macros.h.in "${CMAKE_CURR
 MESSAGE("Build Sayonara ${VAR_SAYONARA_VERSION}")
 MESSAGE("Will install to ${CMAKE_INSTALL_PREFIX}")
 
+# CMake 3.9 searches full UI-file paths
+SET(CMAKE_AUTOUIC_SEARCH_PATHS ${CMAKE_CURRENT_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/src/GUI/Preferences/Fonts)
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR}/src ${CMAKE_CURRENT_BINARY_DIR}/src/GUI/Playlist/ ${CMAKE_BINARY_DIR}/src/GUI/Preferences .)
+
 ADD_SUBDIRECTORY(src)
 ADD_SUBDIRECTORY(resources)
 
